.gitignore

```diff
@@ -6,47 +6,48 @@
 
 # Java files
 .classpath
+.gradle/
```

Comment:
Why was this (and the others, like `build-idea`, `test-output` and `reports`) moved around? Does this make a difference to gitignores?